lib/windows/seekdialog.py

```diff
@@ -754,6 +777,18 @@ def seekMouse(self, action, without_osd=False, preview=False):
             self.updateProgress(set_to_current=False)
             self.setProperty('button.seek', '1')
 
+    def shouldShowIntroSkip(self):
+        if self.intro:
+            if self._enableIntroSkip and \
+                    int(self.intro.startTimeOffset) <= self.offset <= int(self.intro.endTimeOffset):
+                self.setProperty('show.introSkip', '1')
+                if self._introSkipShownFor > self.SHOW_INTRO_SKIP_BUTTON_TIMEOUT:
+                    self.setProperty('show.introSkip_OSDOnly', '1')
+                self._introSkipShownFor += 1
```

Comment:
Let's set a `<timestamp> + self.SHOW_INTRO_SKIP_BUTTON_TIMEOUT` when it becomes visible and change `show.introSkip_OSDOnly` when we've passed that. There's too much potential for someone in the future to add a call to `shouldShowIntroSkip` and cause the value to increment incorrectly.